src/sender.js

```diff
@@ -20,28 +20,42 @@ function Sender (params) {
     }
   })
 
-  this.subpayments = null
-  this.transfers = null
+  this.subpayments = []
+  this.transfers = []
   this.finalTransfer = null
 }
 
 Sender.prototype.findPath = async function () {
-  await this.pathfinder.crawl()
-  this.subpayments = await this.pathfinder.findPath(
-    this.source_ledger,
-    this.destination_ledger,
-    this.destination_amount)
+  if (this.source_ledger === this.destination_ledger) {
+    // Same ledger transfer
+    // TODO add an execution_condition here whenever we switch to having
+    // the recipient creat the execution_condition_fulfillment
+    // TODO ledger fees?
+    this.transfers.push({
```

Comment:
I think using this variable for intra-ledger transfers is confusing -- its used later to build a list of all transfers.